package.json

```diff
@@ -8,7 +8,7 @@
     "start": "node ./app",
     "dev": "nodemon ./app",
     "lint": "eslint .",
-    "format": "prettier --write '**/*.js'",
+    "format": "prettier --write 'bot/**/*.js'",
```

Comment:
We also should apply prettier to js files outside bot/ directory

Reply:
oops, my bad. corrected